scripts/run-k8s-integration-tests.sh

```diff
@@ -6,6 +6,18 @@ ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd)
 
 export GOPATH="${ROOT}/git-kubo-ci"
 kubeconfig="gcs-kubeconfig/${KUBECONFIG_FILE}"
+DEPLOYMENT_NAME="${DEPLOYMENT_NAME:="ci-service"}"
+
+target_bosh_director() {
```

Comment:
We shouldn't wire in the BOSH stuff to our K8s integration tests. We might want to revisit this whole suite and, instead of running commands via `bosh ssh`, run them through a privileged container that `hostPath` mounts the necessary certs.

Alternatively, we can break this out into its own suite of `etcd-integration-tests` if we're worried about the eventual closing-off of `hostPath` mounting.

Reply:
Ok, we break it into security-tests